clang/lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -9041,257 +9041,65 @@ void CGOpenMPRuntime::emitUserDefinedMapper(const OMPDeclareMapperDecl *D,
     return;
   ASTContext &C = CGM.getContext();
   QualType Ty = D->getType();
-  QualType PtrTy = C.getPointerType(Ty).withRestrict();
-  QualType Int64Ty = C.getIntTypeForBitwidth(/*DestWidth=*/64, /*Signed=*/true);
   auto *MapperVarDecl =
       cast<VarDecl>(cast<DeclRefExpr>(D->getMapperVarRef())->getDecl());
-  SourceLocation Loc = D->getLocation();
```

Comment:
Is the location information propagated? I am wondering if debug info might be lost like what happened in the reduction migration patch.

Reply:
This `SourceLocation` is defined but never used in the original Clang code itself.